astropy/coordinates/tests/accuracy/generate_ref_ast.py

```diff
@@ -1,15 +1,47 @@
+# /// script
+# requires-python = ">=3.10, <3.13"
+# dependencies = [
+#     "starlink-pyast==3.15.4",
+#     "numpy==1.26.4",
+#     "astropy",
+# ]
+# ///
+# above are PEP 723 script metadata for later reference, but they are
+# insufficient to run the script: starlink-pyast doesn't have wheels or proper
+# build time requirements specifications.
+# It must be build from source and without isolation, for instance, as follows
+# pip install 'numpy==1.26.4' 'setuptools==74.1.2'
+# pip install 'starlink-pyast==3.15.4' --no-build-isolation
```

Comment:
Why aren't these shell commands written as
```suggestion
# pip install numpy==1.26.4 setuptools==74.1.2
# pip install starlink-pyast==3.15.4 --no-build-isolation
```

Reply:
quoting is needed in certain shells for `>`, `>=`, `<` and `<=`. Out of habit, I apply it everytime, even here when it's not needed. Do you want me to remove these ?

Reply:
Arguments involving `<` or `>` need to be quoted because otherwise the shell will interpret them as redirection operators, but I am not aware of `==` having such special meaning (outside arithmetic expansion or conditional expressions, but we are not using those here). Does Zsh give `==` some meaning I'm not aware of?

Reply:
No. As I said, quoting is not necessary here, I just always do it so I never forget to.